### ISSUE TITLE
convert to torchscript or torchscript_int8 get a confuse result

### ISSUE THREAD

**Issue by cloveropen** [facebookresearch/d2go#156]
when I follow the blog :https://gilberttanner.com/blog/d2go-use-detectron2-on-mobile-devices
convert to balloon datasets, get a confusion：
![1](https://user-images.githubusercontent.com/47469707/147746205-c48887e9-c308-4666-bfed-26214407f0b7.png)

**Comment by wat3rBro** [comment]
Hey @cloveropen , by confusion do you mean it recognizes human as balloon? This is because the balloon dataset only has one class and the model is fine-tuned from the COCO pre-trained model (it's also related to the hyper parameters of fine tuning), it might recognize things in COCO as balloon.

**Comment by cloveropen** [comment]
Hi @wat3rBro the score is mistake，The highest score corresponds to the wrong category

**Comment by wat3rBro** [comment]
I mean since the model is fine-tuned on balloon dataset, the only category that model can predict is the balloon category.

**Comment by cloveropen** [comment]
@wat3rBro thank U,I will try to train again

**Comment by cloveropen** [comment]
when I convert a jit to pt ,in android the scores length and labels length are not equal,the label length=3,but the scores length is 2352 @wat3rBro

**Comment by wat3rBro** [comment]
@cloveropen I'm not sure which output it is, from the number 2352 = 3 * 28^2, perhaps it's the segmentation mask (28x28 is a common resolution for instance mask) for the 3 detected boxes?

**Comment by cloveropen** [comment]
Mask-RCNN-FBNetV3A
Mask-RCNN-FBNetV3A-dsmask
Mask-RCNN-FBNetV3G-FPN

after conver to pt file ,these 3 are not work well,ie:
Mask-RCNN-FBNetV3A-dsmask  : scoresData.legth: 16464 labelsData.length:21

**Comment by wat3rBro** [comment]
@cloveropen the ratio between lengths are the same (i.e. 3:2352 and 21:16464) compared with your original question, so it looks like the same issue. Like I said, the longest output looks like a segmentation mask, which is indeed one of the output from maskrcnn model. The torchscript maskrcnn model should have 4 outputs, including boxes, classes, scores and masks, the scores you're looking for might be at the other output position.

**Comment by cloveropen** [comment]
@wat3rBro thank u a lot,I will try,and test the result
